.changeset/famous-dryers-sing.md

```diff
@@ -0,0 +1,6 @@
+---
+'@spotlightjs/overlay': minor
+---
+
+- Sentry Store Slices
+- fixes: #773: Added a helper for getting localTrces and removed redundant isTraceLocal helper
```

Comment:
Remember that this is the user-facing side of changelogs, so maybe keep it simpler?

```suggestion
Refactor the data store for Sentry integration with slices
```